packages/strapi-plugin-content-manager/admin/src/components/DraggedField/RemoveWrapper.js

```diff
@@ -1,22 +1,22 @@
 import styled from 'styled-components';
 import getColor from './utils/getColor';
 
+/* eslint-disable indent */
+
```

Comment:
We don't need this eslint-disable if we indent the file this way:

```
const RemoveWrapper = styled.div`
  display: flex;
  flex-direction: column;
  justify-content: center;
  position: absolute;
  top: 0px;
  bottom: 0px;
  right: 0;
  width: 30px;
  text-align: center;
  background-color: ${({ isOverEditBlock, isOverRemove, isSelected }) => {
  return getColor(isOverRemove, isSelected, isOverEditBlock);
}};
  cursor: pointer;

  svg {
    align-self: center;
    color: #b4b6ba;
  }

  ${({ isOverRemove }) => isOverRemove && `
    path {
      fill: #f64d0a;
    }
  `};
`;
```

Reply:
I am not able to apply the correct indentation with prettier. Maybe you can submit a PR that fixes it?

Reply:
Yes sure!